src/sidebar/components/view-switcher.js

```diff
@@ -34,6 +42,7 @@ module.exports = {
     isLoading: '<',
     isWaitingToAnchorAnnotations: '<',
     selectedTab: '<',
+    showViewSwitcher: '<',
```

Comment:
The list of properties in the `bindings` section are _inputs_ to the component, ie. attributes in the list provided to `<view-switcher>` in templates that use the component. `showViewSwitcher` is an internal controller method however, so it doesn't need to be here.